mautic/mautic
# restrict workflows with impact to mautic/mautic

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [x]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:

Some workflows have a check in the steps to ensure they are only ran on repos's from the `mautic` organisation, e.g. the [subtree split workflow](https://github.com/mautic/mautic/blob/05a1669ac11a51062bec5cb639141fcc878d144f/.github/workflows/split-monorepo-in-multi-repo.yml#L33)  
Limiting to org alone is not enough in this case, as it allows e.g. any mirror of this repo to accidentally execute those workflows.
(these workflows may fail due to a missing API key, but better prevent this in a controlled way)

This PR adds those safeguards.
See [the documentation](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-only-run-job-for-specific-repository) for an example



#### Steps to test this PR:



The only way to test this is to create an extra repo in the mautic organisation, and check if those workflows aren't triggered.




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/10894"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
